### PR TITLE
Refactor tests utils

### PR DESCRIPTION
**Pull request by evilebottnawi**
Refactor tests and test runner (based on prettier code).

https://github.com/prettier/prettier/blob/master/tests_config/run_spec.js

Now it is better reading and looks.

### CODE REVIEW

File: src/clean.js [hunk 1]
```diff
@@ -34,6 +34,14 @@ function clean(node, newObj) {
     delete newObj.isDoubleQuote;
   }
 
+  if (node.kind === "inline") {
+    if (node.value.includes("___PSEUDO_INLINE_PLACEHOLDER___")) {
+      return null;
+    }
+
+    newObj.value = newObj.value.replace(/\r\n?|\n/g, "");
+  }
```
Comment (czosel) [inline src/clean.js]: Why do we need this?

**Comment by czosel** [comment]
I like the new output :+1:

File: src/clean.js (hunk 1 above)
Comment (evilebottnawi) [reply src/clean.js]: @czosel workaround in `newlines` for `inline` nodes, we can fix it after implement better ast for inline

**Comment by czosel** [comment]
@evilebottnawi can you rebase?

File: src/clean.js (hunk 1 above)
Comment (czosel) [reply src/clean.js]: Alright! Was just a bit confused because this seems unrelated to the test util refactor.

**Comment by evilebottnawi** [comment]
@czosel done :+1:

**Pull request merged**